core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -72,7 +72,7 @@ oppia.directive('collectionNodeCreator', [
           };
 
           var isValidSearchQuery = function(searchQuery) {
-            if (/^[a-zA-Z0-9- ]*$/.test(searchQuery)) {
+            if (/^[a-zA-Z0-9-_ ]*$/.test(searchQuery)) {
```

Comment:
Should spaces be accepted?

Reply:
(Also is this just the regex for the exp ID? Or does it include "search by title" as well?)

Reply:
Yes! for exploration title having spaces e.g. "welcome to oppia!"
![screenshot from 2018-01-12 04 31 44](https://user-images.githubusercontent.com/16653571/34851769-8d5c2014-f751-11e7-9d6e-97aa657097a2.png)

Reply:
Ah ok. Looks good then. Thanks!